edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc

```diff
@@ -13,15 +13,18 @@ You update this CR to specify the image repository of the seed image and to move
 
 include::modules/cnf-image-based-upgrade-prep.adoc[leveloffset=+1]
 
-////
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-creating-backup-resources-with-ztp_ztp-gitops[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent]
-////
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.adoc#cnf-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent]
 
 include::modules/cnf-image-based-upgrade-with-backup.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#ztp-image-based-upgrade-rollback_cnf-image-based-upgrade[(Optional) Moving to the Rollback stage of the image-based upgrade with Lifecycle Agent]
```

Comment:
No error here, but I am surprised to see modules with `(Optional)` prepended to their titles in the docs. That shouldn't be making it through reviews.

Reply:
I will need to open a post-GA PR to add a few comments, I'll make sure to update these.